triton_backend/all_models/tests/test_llmapi_python_backend.py

```
@@ -43,6 +46,7 @@
                      get_streaming_from_request)
 # Use PYTHONPATH=../llmapi/tensorrt_llm/1/
 from model import *
+from model import MultimodalContext, TritonPythonModel  # explicit: avoids F405
```

Ruff (0.16.3)

[error] Module level import not at top of file

(E402)